style.css

```diff
@@ -0,0 +1,88 @@
+#header {
+  display: flex;
+  background-color: #3c3a39;
+  width: 400px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+#owner_name p {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  color: #fff;
+}
+
+#owner_name {
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
+
+#header_menu,
+#owner_name {
+  padding: 0 1rem;
+}
+
+#headline {
+  background: #1c1a19;
+  width: 400px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 5px;
+  padding-bottom: 101px;
+}
+
+#introduction {
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 10px;
+  width: 300px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+}
+
+#Occupation {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  color: #fff5e1;
+}
+
+#qualification {
+  width: 300px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+#contact {
+  width: 100%;
+}
+
+#contact_infos {
+  width: 200px;
+  list-style: none;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 15px;
+}
+
+#illustration {
+  position: absolute;
+  width: 406.89px;
+  height: 312.03px;
+  left: 260px;
+  top: 200px;
+}
```

Comment:
- [ ] As stated the project requirement You must stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). So Kindly Adjust the alignment of the hamburger menu for mobile screen size , Minimum mobile screen size should be 375px.

Your Project at mobile screen size of 375px Look likes this : The Hamburger menu is not visible 
 
![Capture](https://user-images.githubusercontent.com/93598189/211818373-c84d00cb-512a-4ff9-814c-bc2c99adcd09.PNG)

The Figma design Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/211818649-49f65fb0-b059-4e19-af20-ab26ddfaf1c4.PNG)